channels/ON-CHAIN.md

```diff
@@ -96,6 +96,12 @@ the transaction might end up being stuck in the mempool for an extended period.
 The `fee` and `nonce` refer to the `initiator` account, i.e. the `fee` MUST be
 taken from their balance and the `nonce` of their account MUST be incremented.
 
+#### Generalized accounts
+
+If either of the participants is a [Generalized
+account](../generalized_accounts/generalized_accounts.md) the channel create
+will also create an extra entry in the contract state tree, containing a frozen
+authentication state that will be used for off-chain signing and verification.
```

Comment:
I feel this sentence may benefit from being clarified. If "signing and verification" is only off-chain (i.e. not on-chain not even force progress transaction), why including this additional state on-chain? Or maybe this is needed for some processing on-chain? If so, it would help the reader specifying that e.g.
```suggestion
authentication state that will be used for signing and verification in off-chain updates (potentially eventially enforced on-chain).
```

Reply:
Once the off-chain state is brought on-chain it needs to be verified on-chain - it is not clear, thanks!

Reply:
I see this paragraph clarified in new commit. Thanks.